packages/terra-table/src/subcomponents/ColumnHeader.jsx

```diff
@@ -141,13 +140,15 @@ const ColumnHeader = (props) => {
         className={cx('column-header-row', { hidden: !hasVisibleColumnHeaders })}
         height={hasVisibleColumnHeaders ? headerHeight : undefined}
       >
+
         {columns.map((column, columnIndex) => (
           <ColumnHeaderCell
             key={`${column.id}-headerCell`}
             id={`${column.id}-headerCell`}
             tableId={tableId}
             columnId={column.id}
             columnIndex={columnIndex}
+            columnSpan={column.columnSpan}
```

Comment:
Adding the span attribute to the <col> element of the table would eliminate the need to pass this information down as a prop to the ColumnHeader and ColumnHeaderCell components.

Reply:
We are passing all column parameters as props the same way as here. Why do we need to do it different for columnSpan?